frontend/src/lib/functions.ts

```diff
@@ -9,3 +9,26 @@ export function matchAny(search: string, options: string[]): boolean {
 export function capitalize(s: string): string {
     return s[0].toUpperCase() + s.slice(1);
 }
+
+export function convertUnixDate(unixDate: number): string {
+    const date = new Date(unixDate);
+
+    const year = `${date.getFullYear()}`;
+    const month = `0${date.getMonth()}`.slice(-2);
+    const day = `0${date.getDate()}`.slice(-2);
+
+    const hour = `0${date.getHours()}`.slice(-2);
+    const minute = `0${date.getMinutes()}`.slice(-2);
+    const second = `0${date.getSeconds()}`.slice(-2);
+
+    const formatedDate = `${year}-${month}-${day} ${hour}:${minute}:${second}`;
+    return formatedDate;
+}
+
+export function convertValueToMoney(value: number): string {
+    return `£${new Intl.NumberFormat().format(Math.round(value * 100) / 100)}`;
```

Comment:
Just a note: since all our values are actually in USD, may want to use “$” here :)
(I wasn’t 100% sure when I did the initial layout)